WcaOnRails/config/locales/nl.yml

```diff
@@ -3371,14 +3371,9 @@ nl:
       title: Deadline voor het aankondigen van de wedstrijd
       #original_hash: 305cfb1
       content_html: >-
-        De wedstrijd zou minstens een maand van tevoren moeten worden
+        De wedstrijd moet minstens 28 dagen van tevoren worden
```

Comment:
It's best to not let a human change these, because the `original_hash`-es need to be updated as well, or else they will still show up in Jonatan's internationalize app as "in need of re-translating".